tests/boot/boot_from_pxe.pm

```diff
@@ -47,7 +47,7 @@ sub run {
         my $path       = "/mnt/openqa/repo/${image_name}/boot/${arch}/loader";
         my $openqa_url = get_required_var('OPENQA_URL');
         $openqa_url = 'http://' . $openqa_url unless $openqa_url =~ /http:\/\//;
-        my $repo = $openqa_url . "/assets/repo/${image_name}";
+        my $repo = get_required_var('OPENQA_URL') . "/assets/repo/${image_name}";
```

Comment:
I don't see why you change this. Definitely it should not be hidden with the same commit.

Reply:
I will check with this change